api/Makefile

```diff
@@ -129,6 +129,10 @@ start-db:
 ## Destroy current DB, setup new one
 db-recreate: clean-volumes init-db
 
+# TODO
+start-localstack:
+	docker-compose up --detach localstack
+
```

Comment:
Maybe add a brief comment on what we use localstack for

Reply:
Also just something in the docs that explains how to spin this up and test grabbing a file locally?

Reply:
Also should this command automatically get called on `make start` when you're in local?

Reply:
This was just a prototype - would add docs/connect properly in the makefile if I was going to merge this.